projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1276,62 +1428,71 @@ static function revisions_meta_box( $safecss_post ) {
 
 		$posts_per_page = $show_all_revisions ? $max_revisions : 6;
 
-		$revisions = new WP_Query( array(
-			'posts_per_page' => $posts_per_page,
-			'post_type' => 'revision',
-			'post_status' => 'inherit',
-			'post_parent' => $safecss_post['ID'],
-			'orderby' => 'date',
-			'order' => 'DESC'
-		) );
+		$revisions = new WP_Query(
+			array(
+				'posts_per_page' => $posts_per_page,
+				'post_type'      => 'revision',
+				'post_status'    => 'inherit',
+				'post_parent'    => $safecss_post['ID'],
+				'orderby'        => 'date',
+				'order'          => 'DESC',
+			)
+		);
 
-		if ( $revisions->have_posts() ) { ?>
-			<ul class="post-revisions"><?php
+		if ( $revisions->have_posts() ) {
+			?>
+			<ul class="post-revisions">
+			<?php
 
 			global $post;
 
 			while ( $revisions->have_posts() ) :
 				$revisions->the_post();
 
-				?><li>
+				?>
+				<li>
 					<?php
-						echo wp_post_revision_title( $post );
+						echo wp_post_revision_title( $post ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Weird indentation here.
```suggestion
					echo wp_post_revision_title( $post ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```
Or you could add a tab to lines 1457–1459, either way would be fine with me.